osu.Android.props

```
@@ -1,61 +1,24 @@
 <Project>
   <PropertyGroup>
-    <LangVersion>8.0</LangVersion>
-    <OutputPath>bin\$(Configuration)</OutputPath>
-    <WarningLevel>4</WarningLevel>
-    <SchemaVersion>2.0</SchemaVersion>
-    <BundleAssemblies>false</BundleAssemblies>
-    <AotAssemblies>false</AotAssemblies>
-    <OutputType>Library</OutputType>
-    <FileAlignment>512</FileAlignment>
-    <GenerateSerializationAssemblies>Off</GenerateSerializationAssemblies>
-    <AndroidApplication>True</AndroidApplication>
+    <SupportedOSPlatformVersion>21.0</SupportedOSPlatformVersion>
+    <RuntimeIdentifiers>android-x86;android-arm;android-arm64</RuntimeIdentifiers>
+    <AndroidPackageFormat>apk</AndroidPackageFormat>
+    <MandroidI18n>CJK;Mideast;Rare;West;Other;</MandroidI18n>
     <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
```

Works on my device in debug and release when using [`SocketsHttpHandler`](https://github.com/ppy/osu-framework/blob/e6f58ba809a547842b01fc15c00b12b81dd8f09f/osu.Framework/IO/Network/WebRequest.cs#L152-L160) (the default).

https://discord.com/channels/188630481301012481/188630652340404224/966403406707195965

```suggestion
    <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidMessageHandler</AndroidHttpClientHandlerType>
```

---

This should have no effect when using `SocketsHttpHandler`. Platform handler is only used by `HttpClientHandler`. Could you confirm that this changes behavior on yours?